peps/pep-0788.rst

```diff
@@ -0,0 +1,603 @@
+PEP: 788
+Title: Reimagining native threads
+Author: Peter Bierma <zintensitydev@gmail.com>
+Sponsor: Victor Stinner <vstinner@python.org>
+Discussions-To: Pending
+Status: Draft
+Type: Standards Track
+Created: 23-Apr-2025
+Python-Version: 3.15
+Post-History: `10-Mar-2025 <https://discuss.python.org/t/a-new-api-for-ensuring-releasing-thread-states/83959>`__
+
+
+Abstract
+========
+
+:c:func:`PyGILState_Ensure`, :c:func:`PyGILState_Ensure`, and other related
+functions in the ``PyGILState`` family, are the most common way to create
+native threads that interact with Python, and have been the standard for over
+twenty years (:pep:`311`). But, as Python has grown, these functions have
```

Comment:
"as Python has grown" is vague. Perhaps just "over time, problems have become apparent:"?